bin/buildrc

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# enable pattern lists like +(...|...)
+shopt -s extglob
+
+# The master and develop branches are continuously deployed to production and staging.
+# You can also add additional branches here which will be deployed to a temporary staging site.
+# Separate branches with a | character. e.g. "+(branch1|branch2)"
+readonly DEPLOY_BRANCHES="+(annual-report)"
+
+# Use the branch name to set jekyll config:
+# - DTA_SITE_BASEURL - used in jekyll for site.baseurl
+# - DTA_SITE_URL - used in jekyll for site.url
+
+export DTA_SITE_BASEURL=""
+
+readonly GITBRANCH="$(git symbolic-ref --short -q HEAD)"
+case "${GITBRANCH}" in
+  master)
+    export DTA_SITE_URL="https://www.dta.gov.au"
+    ;;
+  develop)
+    export DTA_SITE_URL="https://dta.apps.staging.digital.gov.au"
+    ;;
+  ${DEPLOY_BRANCHES})
+    export DTA_SITE_URL="https://dta-website-${GITBRANCH}.apps.staging.digital.gov.au"
+    ;;
+  *)
+    if [[ -n ${CIRCLECI+x} ]]
+    then
+      # Configure jekyll to be statically hosted on circleci.
+      # TODO remove hardcoded parameters as much as possible - but where does 71211972 come from?
+      export DTA_SITE_URL="https://${CIRCLE_BUILD_NUM}-71211972-gh.circle-artifacts.com"
+      export DTA_SITE_BASEURL="/${CIRCLE_NODE_INDEX}${HOME}/${CIRCLE_PROJECT_REPONAME}/_site"
```

Comment:
I think you can replace `${CIRCLE_NODE_INDEX}${HOME}/${CIRCLE_PROJECT_REPONAME}` with `$CIRCLE_ARTIFACTS`. Or at least that's what I did when I tried this: https://github.com/AusDTO/dto-design-guide/pull/26/files#diff-32d9582cfc1864479f16a9482c7ba5ecR22